com.maplerain.telsis_translator_flutter.yml

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.maplerain.telsis_translator_flutter
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: telsis_translator_flutter
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  # telsis_translator_flutter
+  - name: telsis_translator_flutter
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
```

Comment:
if you set this to `type: archive` it is automatically extracted.

Reply:
I was following an example I found for another flutter-based app on how to package it.
Specifically, this one [here](https://github.com/flathub/codes.merritt.Nyrna).
Will it work okay? Or should I find a way to put all the build commands from the script into the main YAML file?

Reply:
it was done post review. I would have had the same comments. This script is excessiely complex for no reason. It's max 5 lines of `build-commands`

Reply:
Okay, I kind of understand what you mean. I will use [this](https://github.com/flathub/com.github.tglima.Myfluttercrypto) as an example to amend the YAML file and get rid of the build script.